src/pages/workspaces/workspace/workflows/EntitySelectionType.js

```diff
@@ -2,5 +2,7 @@ export default {
   processAll: Symbol('process all'),
   processMergedSet: Symbol('process merged set'),
   chooseRows: Symbol('choose rows'),
-  chooseSets: Symbol('choose sets')
+  chooseSets: Symbol('choose sets'),
+  chooseSetComponents: Symbol('choose set components'),
+  processAllSetComponents: Symbol('process all set components')
```

Comment:
What if we renamed this to `processAllAsSet` or something? (just for clarity...)
Also, am I stirring up a whole thing that's already been figured out if I ask why this option is needed at all?

Reply:
Sounds good to us. It needs to be there because otherwise it's really difficult to select all of the sub entities - the select all check button only selects what's being shown on the page. Does that make sense?

Reply:
Ah. So if you click the little triangle next to the select page checkbox, it'll let you pick just this page or everything, but if you didn't discover that, the user probably hasn't either, so NVM.

Reply:
Ha! Definitely didn't see that. We assumed there was a reason you could select all entities to make a set (for non-set entities, e.g. when you select data for samples, there's a run on all and make a set from it option), and that it was this reason... does that parallelism make sense?